elyra/cli/pipeline_app.py

```diff
@@ -138,8 +137,8 @@ def _preprocess_pipeline(pipeline_path: str,
         raise click.ClickException(f"Error pre-processing pipeline: \n {e}")
 
     if not _validate_pipeline_runtime(primary_pipeline, runtime):
-        runtime_description = primary_pipeline['app_data']['properties']['runtime']
-        runtime_config_display_name = _get_runtime_display_name(runtime_config)
+        runtime_description = _get_runtime_display_name(primary_pipeline['app_data']['runtime'])
```

Comment:
Could we at least change this variable name?  It's not a description at all.  [My previous suggestion](https://github.com/elyra-ai/elyra/pull/2050#discussion_r690423614) used `pipeline_runtime_name` and `requested_runtime_name`.